src/lib/forms/RichEditor.js

```diff
@@ -34,14 +34,14 @@ export class RichEditor extends Component {
       editorConfig,
       inputValue,
       onEditorChange,
-      onInit,
     } = this.props;
     const config = {
       branding: false,
       menubar: false,
       statusbar: false,
       min_height: minHeight,
-      content_style: "body { font-size: 14px; }",
+      content_style:
+        "body { font-size: 14px; " + (disabled ? " opacity: 0.5;" : "") + "}",
```

Comment:
```suggestion
        "body { font-size: 14px; " + (disabled ? "opacity: 0.5; " : "") + "}",
```

most minor nit ever, feel free to ignore